Userland/Utilities/test-crypto.cpp

```diff
@@ -2992,7 +2992,28 @@ static void bigint_signed_bitwise()
         I_TEST((Signed BigInteger | Bitwise or handles sign));
         auto num1 = "-1234567"_sbigint;
         auto num2 = "1234567"_sbigint;
-        if (num1.bitwise_or(num2) == num1) {
+        if (num1.bitwise_or(num1) == num1 && num1.bitwise_or(num2) == num1 && num2.bitwise_or(num1) == num1 && num2.bitwise_or(num2) == num2) {
```

Comment:
Please add new tests to Tests/LibCrypto, test-crypto is not being automatically run anymore.